clang/src/bindgen_ast_consumer.cpp

```diff
@@ -50,11 +50,9 @@ BindgenASTConsumer::~BindgenASTConsumer() {
 
 void BindgenASTConsumer::HandleTranslationUnit(clang::ASTContext &ctx) {
 	this->m_matchFinder.matchAST(ctx);
-// clang segfaults in 6 or newer when calling ParseAST in destructor
-#if __clang_major__ < 6
```

Comment:
Had to disable this for a spec test (macros), it may need to still be resolved for the use case described